css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,19 @@
+h1,
+h2 {
+  font-weight: bolder;
+}
+
+button:hover {
+  background-color: black;
+  color: white;
+  cursor: pointer;
+}
+
+th {
+  background-color: black;
+  color: white;
+}
+
+tr.grey {
+  background-color: lightgrey;
+}
```

Comment:
This should  be accomplished using the `nth-child` pseudo class rather than an actual class